src/app/feed/info-box/info-box.component.ts

```diff
@@ -71,7 +71,11 @@ export class InfoBoxComponent implements OnChanges {
 				}
 			}
 			this.topHashtags = this.parseRawDataWithFrequency(stored);
-			this.areTopHashtagsAvailable = true;
+			if ( this.topHashtags.length === 0 ) {
```

Comment:
- Keeping the code consistent, there should be no space after `(` and before `)`.
  ```js
  if (this.topHashTags.length === 0) {
    ...
  }
   ```

- This can be more simple - 
  ```js
  this.areTopHashtagsAvailable = this.topHashtags.length > 0;
  ```